### PR TITLE
Corrected crash from F8MapView

### PR DESCRIPTION
**Pull request by jpros**
When the user accesses for the first time without connection, and opens the Map tab, the app crashes and never recover (even with internet).

### CODE REVIEW

**Comment by jpros** [comment]
@facebook-github-bot done!

**Comment by qpowell** [comment]
@jpros Thanks. I had to uninstall the app, relaunch it, and avoid clicking on maps in order to check out the rest of it :)

**Comment by frantic** [comment]
@jpros are you running the Parse Server locally?

**Comment by jpros** [comment]
@frantic Yes.

**Comment by frantic** [comment]
@jpros – how about `adb reverse tcp:8080 tcp:8080`?

**Comment by jpros** [comment]
@frantic I`m not understanding your point. There was a bug in the app, and I fixed it.

**Comment by frantic** [comment]
@jpros thanks for you efforts. For me to merge this in, please address the inline comments.

File: js/tabs/maps/F8MapView.js [hunk 1]
```diff
@@ -48,19 +48,33 @@ class F8MapView extends React.Component {
...
       <View style={styles.container}>
         <ListContainer
           title="Maps"
           backgroundImage={require('./img/maps-background.png')}
           backgroundColor={'#9176D2'}>
           <PureListView
-            title={map1.name}
-            renderEmptyList={() => <MapView map={map1} />}
+            title={map1Name}
+            renderEmptyList={() => readerEmptyListMap1}
```
Comment (frantic) [inline js/tabs/maps/F8MapView.js]: `<MapView />` gracefully handles the case when map is empty, we don't need `readerEmptyListMap1/2`.

File: js/tabs/maps/F8MapView.js [hunk 2]
```diff
@@ -48,19 +48,33 @@ class F8MapView extends React.Component {
...
+  	}
+	    
     return (
       <View style={styles.container}>
         <ListContainer
           title="Maps"
           backgroundImage={require('./img/maps-background.png')}
           backgroundColor={'#9176D2'}>
           <PureListView
-            title={map1.name}
```
Comment (frantic) [inline js/tabs/maps/F8MapView.js]: The simplest fix is to just put "Overview"/"Developer Garage" here instead of `map1/2.name`.

**Comment by jpros** [comment]
@frantic Nice, I'll update this. 
Unfortunately I'm new to React Native, this is my first experience with it. As I don't know the F8 project by its entirely, I wasn't sure if the map name was going to be different than "Overview"/"Developer Garage" or not. Also, I didn't know that MapView could handle the case with empty map.

Thanks for your help, and I'm sure there will be a lot of learning about React Native on the next few weeks.

**Comment by frantic** [comment]
Thanks!

File: js/tabs/maps/F8MapView.js (hunk 2 above)
Comment (mtrojan21) [inline js/tabs/maps/F8MapView.js]: Mtrojan21

**Pull request merged**